src/components/notifications/ui-notification-toast.tsx

```diff
@@ -89,9 +89,9 @@ export const UiNotificationToast: React.FC<UiNotificationProps> = ({
   )
 
   const contentDom = useMemo(() => {
-    return content.split('\n').map((value) => {
+    return content.split('\n').map((value, lineNumber) => {
       return (
-        <Fragment>
+        <Fragment key={lineNumber}>
```

Comment:
Same here. Additionally doesn't this break if there are more than one notification on the screen at the same time?

Reply:
s.o.